app/helpers/application_helper.rb

```diff
@@ -260,6 +244,18 @@ def view_to_url(view, parent = nil)
     end
   end
 
+  def class_to_db(cname, parent)
+    return unless cname
+    case
```

Comment:
@jrafanie that is how the original code had it, so this refactoring may have changed behavior without realizing it